### PR TITLE
Initial Ledger Docs

### PR DESCRIPTION
**Pull request by nadinequinr3**
* Rewrite of Key Concepts from C4
* Add UTXO CorDapp page
Preview here: https://docs.preview.docs.r3.com/en/platform/corda/5.0-beta/developing/ledger/ledger.html

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM - clean merge

File: content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    identifier: corda-5-beta-tutorial-deploy-cordapp-utxo
+    weight: 9000
+section_menu: corda-5-beta
+---
+
+UTXO ledger CorDapps should be split in to two parts:
+
+* A contract CPK that contains all code defining the states and the smart contract ruling the creation, evolution, and consumption of the states. The contract CPK needs to be marked as such and can only have dependencies on other contract CPKs, so that it can be loaded into a verify sandbox.
+
+* Workflow CPKs that contain all other code (workflows, persistence, business logic, etc) that can depend on any other CPK, including the contract CPKs.
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md:16]: I never used "etc." before R3 so definitely happy to discuss 👍

File: content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+All of these CPKs can be compiled into one CPB that has both workflows and contracts. A UTXO ledger CPB must also include a notary client and be configured to use a notary. <!--add link-->
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md:18]: @amiegrace12 could you either add a link to the Notary landing page or tell me the path? Thanks

File: content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Finding Transactions and States
+The ledger service has methods to find states (unconsumed, relevant states of a specific type). In Corda 5 beta 1, no further specialised queries are possible, so you must load the state of the required type and filter in the flow code. Custom queries will be available in a future release.
+It is also possible to load transactions by ID.
+
+### Finality
+To finalize a transaction (verify, store, distribute to participants, collect signatures, notarize, distribute signatures), you must call the finalize function on the ledger service. On the counterparty (initiated flow), you need a matching call to `receiveFinality`. These calls will start subflows under the hood that deal with the messaging and finality.
+
+{{< note >}}
+In Beta 1, it is strongly recommended that you wrap the call to `finalize`/`receiveFinality` in a `try...catch` block, and in the case of an exception, do the following:
+* log the exception
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md:56]: I will rewrite

File: content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* suppress the exception
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md:57]: I will rewrite

File: content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+* jump to the end of the flow to stop any further processing
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/developing/ledger/cordapp-utxo.md:58]: I will rewrite

**Review by yarivus — APPROVED** [review]
LGTM

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
Thanks @nadinequinr3 / @paulmoloneyr3 / @colin-hillis

**Pull request merged**